src/plugins/Network.js

```diff
@@ -37,6 +37,9 @@ module.exports = class Network extends OverlayPlugin {
 
         this.androidVersion = "";
 
+        this.wifiInputStatus = true;
+        this.mobileInputStatus = true;
```

Comment:
```suggestion
        this.wifiInputChecked = true;
        this.mobileInputChecked = true;
```
I think this could be clearer, otherwise it doesn't really mean anything to me to have a status being true or false.